app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,14 @@
+<% provide :title, t("about") %>
+<h1><%= t "about" %></h1>
+<p>
+  <%= t "the"%>
+  <%= link_to t("title_book"),Settings.title_book_link
+  %>, <%= t".text6" %>
+  <%= link_to  t("le"), Settings.learn_enough_link %>
+  <% t ".text1" %>, <%= t ".text2" %>
```

Comment:
sao dòng này k cho dùng chung, đặt nhãn sao cho dễ hiểu e nhé, sau maintain toàn text1 textn như thế này tìm đến bao giờ